man/systemd-cryptenroll.xml

```diff
@@ -310,7 +310,9 @@
         <filename>/dev/hidraw1</filename>). Alternatively the special value <literal>auto</literal> may be
         specified, in order to automatically determine the device node of a currently plugged in security
         token (of which there must be exactly one). This automatic discovery is unsupported if
-        <option>--fido2-device=</option> option is also specified.</para>
+        <option>--fido2-device=</option> option is also specified. Note that currently FIDO2 devices
+        enrolled without an accompanying LUKS2 token (i.e. <option>--fido2-token-parameters=false</option>)
```

Comment:
doesn't matter much, but throughout our docs we usually say "yes"/"no" rather than "true"/"false" for booleans, even if we accept both. Hence, maybe change this to `--fido2-token-parameters=no`